omod/src/main/java/org/openmrs/module/locationbasedaccess/fragment/controller/manageAccount/ViewUserLocationFragmentController.java

```diff
@@ -0,0 +1,29 @@
+package org.openmrs.module.locationbasedaccess.fragment.controller.manageAccount;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openmrs.Location;
+import org.openmrs.User;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.locationbasedaccess.LocationBasedAccessConstants;
+import org.openmrs.ui.framework.fragment.FragmentConfiguration;
+import org.openmrs.ui.framework.fragment.FragmentModel;
+
+public class ViewUserLocationFragmentController {
+
+    public void controller(FragmentModel model, FragmentConfiguration config) {
+        Object userUuid = config.getAttribute("userUuid");
```

Comment:
Can we name it as userId instead of userUuid?

Reply:
Changes to userId.